CI実行時にtestでエラーが発生する

https://github.com/chaintope/tapyrus-explorer/actions/runs/26552080208/job/80464492233

テストで Chromeが見つからないエラーが発生している

```
10 06 2026 10:43:56.936:INFO [karma-server]: Karma v6.4.4 server started at http://localhost:9876/
10 06 2026 10:43:56.937:INFO [launcher]: Launching browsers ChromeHeadlessCI with concurrency unlimited
10 06 2026 10:43:56.940:INFO [launcher]: Starting browser ChromeHeadless
10 06 2026 10:43:56.955:ERROR [launcher]: Cannot start ChromeHeadless
	Can not find the binary /home/runner/.cache/puppeteer/chrome/linux-146.0.7680.76/chrome-linux64/chrome
	Please set env variable CHROME_BIN
10 06 2026 10:43:56.955:ERROR [launcher]: ChromeHeadless stdout: 
10 06 2026 10:43:56.955:ERROR [launcher]: ChromeHeadless stderr: 
Error: Process completed with exit code 1.
```

--- a/.github/workflows/ci_on_push_pull_request.yml
+++ b/.github/workflows/ci_on_push_pull_request.yml
@@ -7,7 +7,8 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        node-version: [22.x, 24.x]
+        # Node 24.16.0 は extract-zip の不具合で Puppeteer の Chrome 展開に失敗するため 24.15.0 に固定する (puppeteer/puppeteer#15071)
+        node-version: [22.x, 24.15.0]
     env:
       TZ: "Asia/Tokyo"
     steps:

